crates/db_schema/src/source/oauth_provider.rs

```diff
@@ -0,0 +1,159 @@
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub userinfo_endpoint: Option<DbUrl>,
+  /// The OAuth 2.0 claim containing the unique user ID returned by the provider. Usually this
+  /// should be set to "sub".
+  pub id_claim: Option<String>,
+  /// The OAuth 2.0 claim containing the user name returned by the provider. This depends on the
+  /// provider and could be "name", "username", ...
+  pub name_claim: Option<String>,
+  /// The client_id is provided by the OAuth 2.0 provider and is a unique identifier to this
+  /// service
+  pub client_id: String,
+  /// Lists the scopes requested from users. Users will have to grant access to the requested scope
+  /// at sign up.
+  pub scopes: String,
+  /// Automatically sets email as verified on registration
+  pub auto_verify_email: Option<bool>,
+  /// Automatically approves user application on registration
+  pub auto_approve_application: Option<bool>,
+  /// Allows linking an OAUTH account to an existing user account by matching emails
+  pub account_linking_enabled: Option<bool>,
+  /// switch to enable or disable an oauth provider
+  pub enabled: Option<bool>,
+  pub published: Option<DateTime<Utc>>,
+  pub updated: Option<DateTime<Utc>>,
+}
+
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "full", derive(Insertable, AsChangeset, TS))]
+#[cfg_attr(feature = "full", diesel(table_name = oauth_provider))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct OAuthProviderInsertForm {
+  pub id: OAuthProviderId,
+  pub display_name: String,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub issuer: DbUrl,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub authorization_endpoint: DbUrl,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub token_endpoint: DbUrl,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub userinfo_endpoint: DbUrl,
+  pub id_claim: String,
+  pub name_claim: String,
+  pub client_id: String,
+  pub client_secret: String,
+  pub scopes: String,
+  pub auto_verify_email: bool,
+  pub auto_approve_application: bool,
+  pub account_linking_enabled: bool,
+  pub enabled: bool,
+}
+
+#[derive(Debug, Clone)]
+#[cfg_attr(feature = "full", derive(Insertable, AsChangeset, TS))]
+#[cfg_attr(feature = "full", diesel(table_name = oauth_provider))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct OAuthProviderUpdateForm {
+  pub display_name: String,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub authorization_endpoint: DbUrl,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub token_endpoint: DbUrl,
+  #[cfg_attr(feature = "full", ts(type = "string"))]
+  pub userinfo_endpoint: DbUrl,
+  pub id_claim: String,
+  pub name_claim: String,
+  pub client_secret: Option<String>,
+  pub scopes: String,
+  pub auto_verify_email: bool,
+  pub auto_approve_application: bool,
+  pub account_linking_enabled: bool,
+  pub enabled: bool,
+  pub updated: DateTime<Utc>,
```

Comment:
The updated is usually `Option<Option<`

Reply:
Seems like you didnt push these changes.